src/south/OPCUA_DA/OPCUA_DA.class.js

```diff
@@ -162,22 +179,59 @@ class OPCUA_DA extends ProtocolHandler {
     this.reconnectTimeout = null
 
     try {
+      let keyFileContent = ''
```

Comment:
It seems that the the same very similar code is used to manage certificates in different South. 
Should we move this to you he Protocol class or a library?

Reply:
Those files are the same for OPCUA_DA / HA, but they differ on MQTT (where it needs three files for example). Some additional checks for the format of the files may differ in the future depending of the protocol settings.

That's why I put the logic inside the data source anyway.